modules/GO_TOPGO.R

```diff
@@ -0,0 +1,277 @@
+### Load package and set path
+library("topGO")
+library("org.Mm.eg.db")
+library("ggplot2")
+
+
+
+
+#### Load data
+
+## from file
+#data <- data.table::fread("RESULTS_DEG_HNSC.txt")
+#data$GeneID <- substr(data$GeneID, 1, 18)

```

Comment:
Once you don't need them anymore, please clean up unnecessary print statements, comments and empty lines (usually one uses only 1-2 to separate sections). This makes the code a lot more compact and easier to read.

Reply:
Clean up done